.azure-pipelines/main-pipeline.yml

```diff
@@ -172,21 +165,6 @@ stages:
                   vsixFile: '$(Pipeline.Workspace)/$(artifactPath)/$(packagedTask)'
                   extensionVisibility: 'public'
                   extensionPricing: 'free'
-                  extensionVersion: '$(queryVersion.Extension.Version)'
+                  extensionVersion: $(extensionVersion)
                   updateTasksId: false # The task ID stays `60b18503-c6d6-4e4b-a6b2-52fc6fb3d525` (defined in task.json)
                   updateTasksVersion: false # Bumped automatically by our Release Automation bot
-
-              - task: GitHubRelease@1
```

Comment:
The github release is now created by the Release Automation bot, in `release-version-on-merge.yml`.